crates/pallet-executor/src/lib.rs

```diff
@@ -28,11 +28,13 @@ use sp_runtime::RuntimeAppPublic;
 const INVALID_BUNDLE_EQUIVOCATION_PROOF: u8 = 101;
 const INVALID_TRANSACTION_PROOF: u8 = 102;
 const INVALID_EXECUTION_RECEIPT: u8 = 103;
+const INVALID_BUNDLE: u8 = 104;
```

Comment:
I'd create an enum out of these, then you can cast it to `u8`, but preserve nice ergonomics in most places.

Reply:
Hmm, can't feel it's superior to const 🤔 , try convincing me one more time? :D.

Reply:
```rust
#[repr(u8)]
enum InvalidTransactionCode {
    BundleEquivicationProof = 101,
    TrasactionProof = 102,
    ExecutionReceipt = 103,
    Bundle = 104,
}
```

Then when you need it:
```rust
InvalidTransactionCode::Bundle as u8
```

Or `Into<u8>` can be added.

We do this for errors everywhere, this is not a special case as far as I'm concerned and a bunch of distinct constants doesn't look nicer to me.

Reply:
No strong opinion, updated even though I still don't see it significantly better than the const way :P.